packages/dashboard/src/pages/Wallets/components/Balance/index.tsx

```diff
@@ -70,8 +71,11 @@ const ButtonGroup = styled('div')`
     }
 `
 
-export const Balance = memo(({ balance, onSend, onBuy, onSwap, onReceive }: BalanceCardProps) => {
+export const Balance = memo<BalanceCardProps>(({ balance, onSend, onBuy, onSwap }) => {
```

Comment:
Storybook fails with: "Component must be wrapped with <Container.Provider>"

Reply:
You should not use `useWallet` in this component otherwise it cannot be displayed in the Storybook

Reply:
Did you remove the story for Balance? I don't see it in the latest build https://deploy-preview-3290--compassionate-northcutt-326a3a.netlify.app/storybook-static/dashboard/